front/src/layouts/watchers.tsx

```diff
@@ -6,14 +6,15 @@ import { useAccountEffect } from "wagmi";
 
 import { wagmiConfig } from "@/config/wagmi";
 import { AlephService, BEDROCK_MESSAGE } from "@/services/aleph";
-import { useAccountStore } from "@/stores/account";
+import BedrockService from "@/services/bedrock";
+import { useBedrockAccountStore } from "@/stores/bedrockAccount";
 
 type WatchersProps = {
 	children: ReactNode;
 };
 
 export function Watchers({ children }: WatchersProps) {
-	const accountStore = useAccountStore();
+	const accountStore = useBedrockAccountStore();
```

Comment:
not sure we want to rename everything with "bedrock" (later we will have a store for contacts etc).
Having the "feature" + "Store" is better imo to avoid having the whole code being "bedrockX".

Reply:
and this account store will later also have probably user settings and stuff like that, not only the bedrockService.